.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+
+jdk:
+  - oraclejdk8
+  - openjdk8
```

Comment:
I think we also want 9,10 and 11.
For build environments, we want Windows and Linux, which I think we can do in Travis now.  When I first used Travis, I don't think I could do both.

Reply:
Ah, that explains AppVeyor. I was a bit curious about this when I first looked at, I admit. Then I figured it might be a Travis limitation 

So I guess we configure multiple permutations?
OSX | Windows | Ubuntu
JDK8 | JDK 9 | JDK 10 | JDK 11

I think 9 & 10 have already been EOL'd and are unsupported currently.

Reply:
Looks like the Windows image is WIP and does not have the JDKs yet - see the Travis logs